.github/workflows/build.yml

```diff
@@ -9,11 +9,16 @@ on:
       - master
     types: [opened, reopened, synchronize, labeled]
 
+permissions:
```

Comment:
This is for dependabot but I think the devs are used to dependabot failing. This repo is being migrated to tt api so we don't need to change habits now.

Reply:
removed